custom/icds_reports/utils/aggregation_helpers/distributed/agg_gov_dashboard.py

```diff
@@ -147,5 +153,67 @@ def update_queries(self):
         DROP TABLE temp_gov_dashboard;
         """, {}
 
+
+        yield """
+        CREATE TABLE temp_cbe_data AS
+            SELECT
+                ucr.awc_id,
+                ucr.state_id,
+                ucr.date_cbe_organise,
+                ucr.theme_cbe,
+                ucr.count_targeted_beneficiaries,
+                ucr.count_other_beneficiaries,
+                rank() OVER (
+                    PARTITION BY awc_id
+                    ORDER BY date_cbe_organise, submitted_on
+                    )
+
+            FROM "ucr_icds-cas_static-cbe_form_f7988a04" ucr
```

Comment:
nit: should maybe be extracted to a param / constant

Reply:
in fact I think maybe you meant to put `{cbe_ucr_table}`?

Reply:
Ow. I must have forgot to replace it to param after testing.

Reply:
Updated